cs_util/__init__.py

```diff
@@ -25,14 +25,4 @@
 
 from warnings import warn
 
-from importlib_metadata import version
-
-try:
-    _version = version('cs_util')
-except Exception:  # pragma: no cover
-    _version = 'Unkown'
-    warn(
-        'Could not extract package metadata. Make sure the package is '
-        + 'correctly installed.',
-    )
-__version__ = _version
+__version__ = '0.3.4'
```

Comment:
```suggestion
__version__ = "0.3.4"
```

otherwise Black will complain.